docs/sources/panels-visualizations/visualizations/canvas/index.md

```diff
@@ -270,6 +270,19 @@ You can enable infinite panning in a canvas when pan and zoom is enabled. This a
 Infinite panning is an experimental feature that may not work as expected in all scenarios. For example, elements that are not top-left constrained may experience unexpected movement when panning.
 {{< /admonition >}}
 
+### Tooltip options
+
+| Option                        | Description                                                               |
+| ----------------------------- | ------------------------------------------------------------------------- |
+| [Tooltip mode](#tooltip-mode) | Configure whether tooltips appear when hovering over the canvas elements. |
+
+#### Tooltip mode
+
+Controls the display of tooltips when hovering over elements in a canvas that are connected to data, data links, or actions.
```

Comment:
Since there's only one option, let's not bother with the table and take out the h4 heading.

```suggestion
The **Tooltip mode** setting controls the display of tooltips when hovering over elements in a canvas that are connected to data, data links, or actions.
The options are:
```